CHANGELOG.md

```diff
@@ -41,6 +41,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+
+## [v8.0.0 - Emergency proposals upgrade](https://github.com/osmosis-labs/osmosis/releases/tag/v8.0.0)
+
+This upgrade is a patch that must be hard forked in, as on-chain governance of Osmosis approved proposal 227 and proposal 228.
+
+This upgrade includes:
+
+* Adding height-gated AnteHandler message filter to filter unpooling tx pre-upgrade.
+* At block height 4402000 accelerates prop 225, which in turn moves incentives from certain pools according to props 222-224
+* Adds a msg allowing unpooling of UST pools. 
+  * This procedure is initiated by whitelisting pools 560, 562, 567, 578, 592, 610, 612, 615, 642, 679, 580, 635. 
+  * Unpooling allows exiting whitelisted pools directly, finish unbonding duration with the exited tokens instead of having to wait unbonding duration to swap LP shares back to collaterals. 
+  * This procedure also includes locks that were already unbonding pre-upgrade and locks that were superfluid delegated.
+
+Every node should upgrade their software version to v8.0.0 before the upgrade block height 4402000. If you use cosmovisor, simply swap out the binary at upgrades/v7/bin to be v8.0.0, and restart the node. Do check cosmovisor version returns v8.0.0
+
+### Features 
+* {Across many PRs} Initiate emergency upgrade 
+* [#1481] Emergency upgrade as of prop [226] (https://www.mintscan.io/osmosis/proposals/226) 
+* [#1435] `x/tokenfactory` create denom fee for spam resistance 
+* [#1429] solver for multi-asset CFMM
+* [#1253] Add lockup duration edit method
```

Comment:
Please make the numbers link to the PRs similar to earlier releases

Reply:
We actually made a script for this at some point in the SDK, just it never fit in the natural dev flow. I wonder if theres a good way to fit it back in